backend/gateway/mux/mux_test.go

```diff
@@ -122,3 +128,108 @@ func TestCustomHeaderMatcher(t *testing.T) {
 		assert.Equal(t, test.expectedBool, ok)
 	}
 }
+
+func TestCustomErrorHandler(t *testing.T) {
```

Comment:
What about a test case that asserts the query params stay intact?

Reply:
see :157, it checks that the whatever full uri was passed in is preserved

Reply:
Ah I see now, ty.